modules/osdk-scorecard-store-output.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-scorecard.adoc
+
+:osdk_ver: v1.16.0
+
+:_content-type: MODULE
+[id="osdk-scorecard-store-output_{context}"]
+= Local storage of scorecard results
+
+You can enable local storage of scorecard test results by using the `scorecard` command or by modifying your `config.yaml` file.
+
+[IMPORTANT]
+====
+There is a known issue with the Operator SDK scorecard storage utility. Currently, running the `operator-sdk scorecard` command with the `--test-output` flag returns an empty directory. There is no known workaround at this time. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2017669[*BZ#2017669*])
+====
+
+Running the `scorecard` command with the `--test-output` flag stores the scorecard test results in a specified directory. If you do not specify a directory, the command stores the test results in the `test-output` directory of your working directory.
+
+.`scorecard --test-output` command syntax
+[source,terminal,subs="attributes+"]
+----
+$ operator-sdk scorecard <bundle_directory_or_image> --test-output=/<output_directory> <.>
+----
+<.> Define the output directory.
```

Comment:
Honestly, I'd rewrite this whole thing without the broken flag and be like "this command store the test results in the `test-output` directory of your working directory." When the bug is fixed, update the docs to describe the flag.

Reply:
+1 The why this all is written makes this read as if 'we have this cool new feature, but none of it works - good luck'. 

I wouldn't document this till the bug was/is fixed; only document the way's in which it properly works.